### PR TITLE
lint

### PR DESCRIPTION
**Pull request by kiorky**

### CODE REVIEW

**Comment by kiorky** [comment]
this refs https://github.com/saltstack/salt/pull/21314

File: tests/unit/modules/deb_postgres_test.py [hunk 1]
```diff
@@ -73,17 +79,20 @@ class PostgresLsClusterTestCase(TestCase):
...
-                                           'user': 'postgres',
-                                           'datadir':'/srv/8.4/main',
-                                           'log':'/var/log/postgresql/postgresql-8.4-main.log'},
-                          ('9.1', 'main'):{'port':5433,
-                                           'status': 'online',
-                                           'user': 'postgres',
-                                           'datadir':'/srv/9.1/main',
-                                           'log':'/var/log/postgresql/postgresql-9.1-main.log'},},
-                         deb_postgres._parse_pg_lscluster(stdout))
+        self.assertDictEquala(
```
Comment (arthurzenika) [inline tests/unit/modules/deb_postgres_test.py]: s/assertDictEquala/assertDictEqual/

**Comment by arthurzenika** [comment]
thanks for the lint @kiorky (my excuse for not doing it was the rush before ending the sprint)

**Comment by kiorky** [comment]
no problem, just helping a bit here to get this in !

**Pull request merged**